docs/cosmology/index.rst

```diff
@@ -89,6 +93,8 @@ access the floating point or array values::
   (69.32, Unit("km / (Mpc s)"))
 
 
+|
```

Comment:
Too much whitespace?

Reply:
I think uniformity of the docs is important too, though!

Reply:
True. It's probably easier to just wait on the scipy theme change...

Reply:
I've reverted the manual spaces.